Modules/Microsoft365DSC/DSCResources/MSFT_SPOOrgAssetsLibrary/MSFT_SPOOrgAssetsLibrary.psm1

```diff
@@ -376,6 +376,8 @@ function Export-TargetResource
     $i = 1
     $dscContent = ''
 
+    $tenantName = $TenantId.Split(".")[0]
```

Comment:
not sure if this will work for custom domains. your best bet would be to take a look at Get-SPOAdminUrl in the MSCloudLoginAssistant(our) and refactor the main logic into a separate function that gets the root site sharepoint URL.